From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Doe <john.doe@somewhere.on.planet>
Date: Tue, 12 Mar 2024 18:54:10 +0100
Subject: Patching u-boot sunxi64 files drivers/video/videomodes.c

Signed-off-by: John Doe <john.doe@somewhere.on.planet>
---
 drivers/video/videomodes.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/drivers/video/videomodes.c b/drivers/video/videomodes.c
index 35955a5df7..805253d77d 100644
--- a/drivers/video/videomodes.c
+++ b/drivers/video/videomodes.c
@@ -216,10 +216,12 @@ int video_get_params (struct ctfb_res_modes *pPar, char *penv)
 			p++;	/* skip ',' */
 	}
 	return bpp;
 }
 
+static const char *g_options = "monitor=dvi,hdp=0,edid=0";
+
 /*
  * Parse the 'video-mode' environment variable
  *
  * Example: "video-mode=fslfb:1280x1024-32@60,monitor=dvi".  See
  * doc/README.video for more information on how to set the variable.
@@ -237,10 +239,17 @@ int video_get_video_mode(unsigned int *xres, unsigned int *yres,
 {
 	char *p = env_get("video-mode");
 	if (!p)
 		return 0;
 
+	xres = 1920;
+	yres = 1080;
+	depth = 24;
+	freq = 60;
+	*options = g_options; 
+	return 1;
+	
 	/* Skip over the driver name, which we don't care about. */
 	p = strchr(p, ':');
 	if (!p)
 		return 0;
 
-- 
Created with Armbian build tools https://github.com/armbian/build

